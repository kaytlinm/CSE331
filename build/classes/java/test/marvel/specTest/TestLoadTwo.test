#Tests that two independent graphs can be loaded from two independent .tsv files

LoadGraph graph1 smallTest.tsv
LoadGraph graph2 staffSuperheroes.tsv

ListChildren graph1 Bob
ListChildren graph2 Ernst-the-Bicycling-Wizard