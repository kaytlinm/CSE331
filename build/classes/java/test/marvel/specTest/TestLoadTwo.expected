#Tests that two independent graphs can be loaded from two independent .tsv files

loaded graph graph1
loaded graph graph2

the children of Bob in graph1 are: Cole(vol1) Erin(vol4) Sue(vol1)
the children of Ernst-the-Bicycling-Wizard in graph2 are: Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Notkin-of-the-Superhuman-Beard(CSE403) Perkins-the-Magical-Singing-Instructor(CSE331)